client/graphql/queries/userProfileSets.graphql

```diff
@@ -0,0 +1,14 @@
+#import "graphql/fragments/userSets.graphql"
+
+query userProfileSets(
```

Comment:
Maybe we could just make this `userProfile` which gets all the info required for their profile, including any future additions like folders.